nixos/modules/services/web-servers/apache-httpd/wordpress.nix

```diff
@@ -85,10 +85,10 @@ let
       # remove bundled themes(s) coming with wordpress
       rm -Rf $out/wp-content/themes/*
 
-      # symlink additional theme(s)
-      ${concatMapStrings (theme: "ln -s ${theme} $out/wp-content/themes/${theme.name}\n") config.themes}
-      # symlink additional plugin(s)
-      ${concatMapStrings (plugin: "ln -s ${plugin} $out/wp-content/plugins/${plugin.name}\n") (config.plugins) }
+      # hard link additional theme(s)
```

Comment:
Could you change this to "Copy additional theme(s)"?

Same thing for the plugins comment.

Reply:
Done